regparser/commands/tests.py

```diff
@@ -0,0 +1,51 @@
+import click
+import importlib
+import nose
```

Comment:
Nose isn't required in the base package. Would it make sense for this import to happen optionally, or even within the `tests()` functions?